


[0;36m------------------- TESTING game -------------------[0m
Service: game, App: game_app, Database: game
-------------------


[0;33mSaving data in the Python container (game) for app 'game_app'...[0m


[0;36mRunning the Django shell script inside container game...[0m
[0;31mFailed to save data for game_app in Python container game.[0m



[0;33mVerifying data in the database (game) using container (db_game)...[0m
[0;31mFailed to retrieve data for game_app in database game.[0m



[0;33mTesting connectivity to database (game) in container (db_game)...[0m
[0;31mDatabase connectivity test failed for game.[0m




[0;36mAll tests completed![0m
